assets/src/edit-story/theme.js

```diff
@@ -63,12 +68,14 @@ export const GlobalStyle = createGlobalStyle`
   }
 
   *::-webkit-scrollbar-track {
-    background: ${({ theme }) => rgba(theme.colors.bg.black, 0.1)};
+    background: ${({ theme }) =>
+      rgba(theme.DEPRECATED_THEME.colors.bg.black, 0.1)};
```

Comment:
Could we adjust the scrollbar colors here once and for all? They are as you pointed out way too faint. So change this to `theme.colors.bg.tertiary`, which should be a "Gray / 70". And also in the other places necessary to address scrollbars throughout.

Reply:
So, no `rgba(... , .1)` here - just the tertiary gray as-is.

Reply:
I think it's better to do this separately. There are multiple places in the code where scrollbar colors are defined currently and in some places, these seem to differ -- e.g. for the library vs zoom vs popup. So I think it's better not to hold off this PR for confirming and verifying the scrollbar colors -- we can add a separate issue that does that for all the scrollbars.